### PR TITLE
Get User Data

### PR DESCRIPTION
**Pull request by jaeivan**
Made API to get user data given UID and add to trip hook!

### CODE REVIEW

**Comment by jaeivan** [comment]
Postman Test
image

File: utility/hooks/trip.tsx [hunk 1]
```diff
@@ -106,9 +103,11 @@ export function TripProvider({ children }: { children: React.ReactNode }) {
     itinerary: [],
     joinableEvents: [],
     photoURL: "",
+    userData: new Map<string, User>(),
     days: [],
     didReadLayout: false,
   })
+  console.log(trip)
```
Comment (jonathanhuydieu) [inline utility/hooks/trip.tsx]: Oopsies! Looks like you left a console.log()!

**Review by jonathanhuydieu — APPROVED** [review]
Lovely work Senor.

File: utility/hooks/trip.tsx (hunk 1 above)
Comment (jaeivan) [reply utility/hooks/trip.tsx]: 🧢

Comment (Victoria-Williamson) [reply utility/hooks/trip.tsx]: this was actually me...

**Pull request merged**